src/components/chatbot/chatbot.css

```diff
@@ -0,0 +1,80 @@
+.chatbot-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 500px;
+    background-color: #f5f5f5;
+    padding: 20px;
+    padding-bottom: 50px;
+}
+
+.chatbot-form {
+    width: 100%;
+    max-width: 620px;
+    margin-bottom: 20px;
+    padding-right: 22px;
+}
+
+.chatbot-input {
+    width: 100%;
+    height: 25px;
+    padding: 10px;
+    font-size: 16px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    margin-bottom: 10px;
+    resize: none;
+    background-color: #fff;
+    color: #333;
+}
+
+.chatbot-output {
+    width: 100%;
+    max-width: 600px;
```

Comment:
These width 100% max-width set to something else isn't great. You could leave max width but maybe add a wrapper and set that wrapper to that size. We'd have to review the actual running site and see what layout you are trying to achieve here but could be improved most likely